src/components/map/controls/timeline/component.jsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Timestep } from 'vizzuality-components';
+import Icon from 'components/icon';
+
+import styles from './styles.module.scss';
+
+const TimelineComponent = ({
+  activeTab,
+  setActiveTab,
+  className,
+  toggleTimelineSpeed,
+  title,
+  data,
+  selectedSpeed,
+  timelineParams,
+  handleOnChange,
+  hideHeader,
+  hideTimeline,
+  setYearIndex,
+  currentYear
+}) => {
+  return (
+    <div className={cx(styles.container, className)}>
+      {!hideHeader && data && (
+        <div className={styles.header}>
+          <button className={styles.infoButton}>
+            <Icon name="icon-info" />
+          </button>
+          <span className={styles.title}>{title}</span>
+          {Object.keys(data).length > 1 &&
+            Object.keys(data).map(key => (
+              <button
+                id={key}
+                className={cx(styles.tab, { [styles.activeTab]: key === activeTab })}
+                onClick={() => setActiveTab(key)}
+              >
+                {data[key].name}
+              </button>
+            ))}
+        </div>
+      )}
+      {!hideTimeline && (
+        <div className={styles.timeline}>
+          <Timestep
+            {...timelineParams}
+            handleOnChange={dates => {
+              handleOnChange(dates);
```

Comment:
This is what I'm getting from the GraphCMS API, do you want me to filter it manually here anyway?

Reply:
Uhm tricky. Because of a decision we took when we had to move years around (to make 'current' appear on RCP charts), now you get that year in the RCP query (right map). But you have that value in the 'present' map (left), so I'd filter that client-side, yes. What do you think?

Reply:
If we don't wont to have 1995 on the timeline I can filter it on the client-side, I'll do this now :+1:

Reply:
@blayhem done :)